Misc/NEWS.d/next/Build/2021-10-11-16-27-38.bpo-45405.iSfdW5.rst

```diff
@@ -0,0 +1,2 @@
+Prevent ``internal configure error`` when running ``configure``
```

Comment:
can we delete this file?

Reply:
We can. It came along for the ride with the `git cherry-pick` of the fix commit.